confluent_kafka/avro/serializer/message_serializer.py

```diff
@@ -204,6 +204,9 @@ def decode_message(self, message):
 
         if message is None:
             return None
+          
+        if len(message) == 0:
+            return ""
```

Comment:
I don't think this is what the application expects, the deserializer is set up to return a certain deserialized object type - returning an empty string will most likely not be that object.

So I believe the existing behaviour is correct.

Reply:
What I observed is, when the message key is empty string, the input `message` object is just `b''`, so there's no magic byte and schema_id.

So would it be better if check `if message == b''` here?

Reply:
My point is that an AvroConsumer will expect properly serialized messages, any other message should race a deserialization exception, as it does today.

Reply:
ok that's fair.